From df1875d77194af07343ef8378bca41b4b3a84f03 Mon Sep 17 00:00:00 2001
From: Maruthi Srinivas Bayyavarapu <Maruthi.Bayyavarapu@amd.com>
Date: Thu, 25 Jun 2015 18:09:00 +0530
Subject: [PATCH 299/343] drm/amd: remove amd gnb bus default runtime pm ops

with the default gnb bus runtime pm ops, alsa pcm device attached to
it is unable to get runtime suspended/resumed.

Signed-off-by: Maruthi Bayyavarapu <maruthi.bayyavarapu@amd.com>
Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Sanjay R Mehta <sanju.mehta@amd.com>
---
 drivers/gpu/drm/amd/bus/amd_gnb_bus.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/gpu/drm/amd/bus/amd_gnb_bus.c b/drivers/gpu/drm/amd/bus/amd_gnb_bus.c
index 071b16c..13d5506 100644
--- a/drivers/gpu/drm/amd/bus/amd_gnb_bus.c
+++ b/drivers/gpu/drm/amd/bus/amd_gnb_bus.c
@@ -142,11 +142,6 @@ static const struct dev_pm_ops amd_gnb_bus_device_pm_ops = {
 	.thaw = amd_gnb_bus_device_pm_thaw,
 	.poweroff = amd_gnb_bus_device_pm_poweroff,
 	.restore = amd_gnb_bus_device_pm_restore,
-	SET_RUNTIME_PM_OPS(
-		pm_generic_runtime_suspend,
-		pm_generic_runtime_resume,
-		pm_runtime_idle
-	)
 };
 
 /* The bus should only be registered by the first amd_gnb, but further
-- 
1.9.1

